test_unstructured_ingest/test-ingest-sharepoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=Sharepoint
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+
+if [ -z "$MS_CLIENT_ID" ] || [ -z "$MS_CLIENT_CRED" ]; then
```

Comment:
should not check for MS_CLIENT_ID and MS_CLIENT_CRED here. instead we need to validate that: SHAREPOINT_CLIENTID, SHAREPOINT_CRED, and SHAREPOINT_SITE are set